srcpkgs/bird/template

```diff
@@ -1,17 +1,17 @@
 # Template file for 'bird'
 pkgname=bird
-version=2.0.9
-revision=2
-wrksrc=bird-v$version
+version=2.0.10
+revision=1
 build_style=gnu-configure
 hostmakedepends="flex autoconf"
 makedepends="ncurses-devel readline-devel libssh-devel"
 short_desc="BIRD Internet Routing Daemon"
 maintainer="Philipp Hirsch <itself@hanspolo.net>"
 license="GPL-2.0-or-later"
 homepage="https://bird.network.cz"
-distfiles="https://gitlab.labs.nic.cz/labs/bird/-/archive/v${version}/bird-v${version}.tar.gz"
-checksum=912d5c1bbefffd6198b10688ef6e16d0b9dfb2886944f481fc38b4d869ffd2c4
+changelog="https://gitlab.nic.cz/labs/bird/-/raw/master/NEWS"
+distfiles="https://gitlab.labs.nic.cz/labs/bird/-/archive/${version}/bird-${version}.tar.gz"
```

Comment:
Could we use this tarball instead?
```suggestion
distfiles="https://bird.network.cz/download/bird-${version}.tar.gz"
```

Reply:
Yeah, that one seems more consistent historically.